hazelcast/src/test/java/com/hazelcast/topic/impl/reliable/LossToleranceTest.java

```diff
@@ -130,24 +117,19 @@ public void whenLossTolerant_andOwnerCrashes_thenContinue() {
         topic.publish("item1");
         topic.publish("item2");
 
-        assertTrueEventually(new AssertTask() {
-            @Override
-            public void run() {
-                assertContains(listener.objects, "item1");
-                assertContains(listener.objects, "item2");
-            }
+        assertTrueEventually(() -> {
+            assertContains(listener.objects, "item1");
+            assertContains(listener.objects, "item2");
         });
         TestUtil.terminateInstance(topicOwnerInstance);
 
-        topic.publish("newItem");
-
-
-        assertTrueEventually(new AssertTask() {
-            @Override
-            public void run() {
-                assertContains(listener.objects, "newItem");
+        assertTrueEventually(() -> {
+            String item = "newItem " + UUID.randomUUID();
+            topic.publish(item);
+            assertTrueEventually(() -> {
+                assertContains(listener.objects, item);
                 assertFalse(topic.runnersMap.isEmpty());
```

Comment:
I don't know why we assert this bit - as it's internal detail

Reply:
True, I'll remove it.